hs-abci-sdk/src/Tendermint/SDK/Modules/Bank/Types.hs

```diff
@@ -0,0 +1,89 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
there's no need for orphan instances here, move any instance declarations to the module where `Address` is defined